app/src/lib/stores/app-store.ts

```diff
@@ -2903,6 +2905,14 @@ export class AppStore extends TypedBaseStore<IAppState> {
     return Promise.resolve()
   }
 
+  public _setCommitMessageFocus(focus: boolean) {
+    const previousValue = this.focusCommitMessage
+    this.focusCommitMessage = focus
+    if (this.focusCommitMessage !== previousValue) {
```

Comment:
This could be cleaned up a bit to make it clearer that this property should be set only if it changes.

```ts
    if (this.focusCommitMessage !== focus) {
      this.focusCommitMessage = focus
      this.emitUpdate()
    }
```

Reply:
Updated.